### ISSUE TITLE
Started to get error 400 when trying to authenticate

### ISSUE THREAD

**Issue by FlyingDiver** [FlyingDiver/Indigo-BMW-CD#6]
https://forums.indigodomo.com/viewtopic.php?f=216&t=23303&start=60#p197202

```
   Reloading plugin "BMW Connected Drive 0.0.6"
   Stopping plugin "BMW Connected Drive 0.0.6" (pid 863)
   BMW Connected Drive Debug       runConcurrentThread ending
   BMW Connected Drive             BMW Connected Drive Account: Stopping cdAccount Device xxxxxx
   BMW Connected Drive             BMW X1: Stopping cdVehicle Device xxxxxxx
   BMW Connected Drive             Stopping Connected Drive
   Stopped plugin "BMW Connected Drive 0.0.6"
   Starting plugin "BMW Connected Drive 0.0.6" (pid 50532)
   BMW Connected Drive Debug       logLevel = 10
   Started plugin "BMW Connected Drive 0.0.6"
   BMW Connected Drive             Starting Connected Drive
   BMW Connected Drive Debug       updateFrequency = 300.0
   BMW Connected Drive             BMW Connected Drive Account: Starting cdAccount Device
   BMW Connected Drive Error       get_tokens AUTH call failed, response code = 400
   BMW Connected Drive             BMW X1: Starting cdVehicle Device
   BMW Connected Drive Debug       runConcurrentThread starting
   BMW Connected Drive Error       get_tokens AUTH call failed, response code = 400
   BMW Connected Drive Debug       ConnectedDrive account xxxxxxx not authenticated, skipping update
   BMW Connected Drive Debug       ConnectedDrive get_vehicle_data: xxxxxxxx
   BMW Connected Drive Error       Error in plugin execution runConcurrentThread:

Traceback (most recent call last):
  File "plugin.py", line 140, in runConcurrentThread
  File "plugin.py", line 177, in updateVehicle
  File "/Library/Application Support/Perceptive Automation/Indigo 7.4/Plugins/BMW ConnectedDrive.indigoPlugin/Contents/Server Plugin/bmwcdapi.py", line 149, in get_vehicle_data
TypeError: 'NoneType' object has no attribute '__getitem__'
```

**Comment by FlyingDiver** [comment]
Possibly related:
https://github.com/bimmerconnected/bimmer_connected/pull/252/files

**Comment by FlyingDiver** [comment]
Fixed in 0.0.7
